scripts/build_peach_go_sbot.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+"""
+script to create debian packages for cross-compiled go binaries for go-sbot
+based off of this post
+https://unix.stackexchange.com/questions/627689/how-to-create-a-debian-package-from-a-bash-script-and-a-systemd-service
+"""
+
+
+from constants import *
+from utils import render_template
+
+import subprocess
+import argparse
+import sys
+import os
+import shutil
+
+DEB_CONF_DIR = os.path.join(PROJECT_PATH, 'conf/templates/peach_go_sbot')
+DEB_BUILD_DIR = "/tmp/peach_go_sbot"
+GO_SSB_DIR = "/srv/peachcloud/automation/go-ssb"
+
+
+def crosscompile_peach_go_sbot():
+    subprocess.check_call(["git", "pull"], cwd=GO_SSB_DIR)
+    # TODO: confirm that version number in the repo matches the version number we set for the package we are building
```

Comment:
Sounds like a job for Regex :)

Reply:
Just added a github issue for this so we don't forget it https://github.com/peachcloud/peach-vps/issues/11